apps/frontend/src/app/5_pages/AavePage/components/CollateralRatioHealthBar/CollateralRatioHealthBar.utils.tsx

```diff
@@ -6,7 +6,7 @@ export const getCollateralRatioThresholds = (minimum: Decimal) => {
   return {
     START: 100,
     MIDDLE_START: minimumCollateralRatio.toNumber() - 0.1,
-    MIDDLE_END: minimumCollateralRatio.mul(1.2).toNumber(),
-    END: minimumCollateralRatio.mul(1.6).toNumber(),
+    MIDDLE_END: minimumCollateralRatio.mul(1.5).toNumber(),
+    END: minimumCollateralRatio.mul(2).toNumber(),
```

Comment:
Seems like these are configs, in that case i think they should be extracted into a constant.

Reply:
Nice catch, but no need in this case, it's better to hardcode it because it will no chance.